frontend/src/hooks/useMovies.ts

```diff
@@ -28,5 +28,10 @@ export default function useMovies(){
             .then(getAllMovies)
     }
 
+    const deleteMovie = (id:string) => {
+        axios.delete("/api/movie/"+id)
+            .catch((e) => toast.error(e.message))
```

Comment:
Hier fehlt noch ein .then um die Movies neuzuladen

```suggestion
        axios.delete("/api/movie/"+id)
            .then(getAllMovies)
            .catch((e) => toast.error(e.message))
```